compiler/src/iree/compiler/GlobalOptimization/Passes.cpp

```diff
@@ -91,6 +91,9 @@ void buildGlobalOptimizationPassPipeline(
 
   // Preprocessing passes to get the program into a canonical state.
   FunctionLikeNest(mainPassManager)
+      .addPass(createLinalgQuantizedConvToConvPass)
+      .addPass(createLinalgQuantizedMatmulToMatmulPass)
+      .addPass(IREE::Flow::createCanonicalizerPass)
```

Comment:
Not a great idea to have flow pass used every where. Maybe move this to `Util` dialect or something.

Reply:
I think Ian is moving the dispatch region formation pipeline somewhere else and I think this pass will go with it. Let's see where it lands after that move.